packages/extension/src/components/molecules/TokenDisplay/TokenDisplay.tsx

```diff
@@ -45,12 +51,16 @@ export const TokenDisplay: FC<TokenDisplayProps> = ({
       <div style={{ display: 'flex', alignItems: 'center' }}>
         {isXRPToken ? <Xrp /> : <GemWallet />}
         <div style={{ marginLeft: '10px' }}>
-          {/* We display the trustline in brown if its limit is 0 or if the noRipple flag is set to false */}
-          <Typography
-            style={trustlineLimit === 0 || trustlineNoRipple === false ? { color: 'brown' } : {}}
+          <Tooltip
+            title={
+              isTokenWarning ? 'Trustline limit set to 0 or rippling not prevented' : undefined
```

Comment:
I would use two distinct memos and display the correct warning message even though its more code, i believe it's way cleaner for the user